jsonschema/apis/Events_v1_000.json

```diff
@@ -0,0 +1,369 @@
+                    },          
+                    {
+                        "$ref": "https://raw.githubusercontent.com/totvs/ttalk-standard-message/events/v1/1_000/jsonschema/apis/types/totvsApiTypesBase.json#/parameters/Fields"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description":"Operação realizada com sucesso!",
+                        "content":{
+                            "application/json":{
+                                "schema":{                                    
+                                    "$ref":"https://raw.githubusercontent.com/totvs/ttalk-standard-message/events/v1/1_000/jsonschema/schemas/Events_1_000.json#/definitions/PagedEvent"                                    
+                                }
+                            }
+                        }
+                    },
+                    "400": {
+                        "description": "erro no momento da requisição",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "$ref": "https://raw.githubusercontent.com/totvs/ttalk-standard-message/events/v1/1_000/jsonschema/apis/types/totvsApiTypesBase.json#/definitions/ErrorModel"
+                                }
+                            }
+                        }
+                    }
+                }
+            }, 
+            "post":{
+				"tags":[
+					"events"
+				],
+				"summary": "Inclui um novo evento",
+				"description": "Inclui um novo evento",
+				"operationId": "postevents",
+				"x-totvs":{
+					"productInformation":[
+						{
+							"product":"protheus",
+							"available": true,
+							"note": "Inclui um novo evento",
+							"minimalVersion": "12.1.21"
+						}
+					]
+                },
+                "parameters": [
+                    {
+                        "$ref": "https://raw.githubusercontent.com/totvs/ttalk-standard-message/events/v1/1_000/jsonschema/apis/types/totvsApiTypesBase.json#/parameters/Authorization"
+                    },
+                    {
+                        "$ref": "https://raw.githubusercontent.com/totvs/ttalk-standard-message/events/v1/1_000/jsonschema/apis/types/totvsApiTypesBase.json#/parameters/Fields"
+                    }
+                ],
+                "requestBody": {
+                    "content": {
+                        "application/json": {
+                            "schema": {
+                                "$ref": "https://raw.githubusercontent.com/totvs/ttalk-standard-message/events/v1/1_000/jsonschema/schemas/Events_1_000.json#/definitions/Events"
+                            }
+                        }
+                    },
+                    "description": "evento a ser incluido"
+                },
+                "responses" : {
+                    "200": {
```

Comment:
Quanto o POST corresponde a inclusão de dados, o mais indicado é retornar 201 (Created).

Reply:
Item não foi ajustado. Favor justificar, se não for alterar.